lib/datadog/lambda.rb

```diff
@@ -58,5 +68,34 @@ def self.metric(name, value, time: nil, **tags)
       metric = { e: time_ms, m: name, t: tag_list, v: value }.to_json
       puts metric
     end
+
+    def self.gen_enhanced_tags(context)
+      arn_parts = context.invoked_function_arn.split(':')
```

Comment:
I know it's unlikely that the function arn will be invalid in an actual lambda environment, but we should make sure this doesn't crash when a user runs this code in a test environment. It should be possible to call wrap with an invalid context object.

Reply:
Okay, done